.github/README.md

```diff
@@ -1,3 +1,5 @@
+<img src="https://github.com/OneLiteFeatherNET/repo-name/blob/main/readme-assets/logo.png" alt="Microtus Logo" width="200">
```

Comment:
Looks like the logo is not working: 
![image](https://github.com/user-attachments/assets/9c6e67aa-d941-4f87-ac49-5e645eb4048c)

Also, `repo-name` is a placeholder here

Reply:
Great work on this contribution! I appreciate the effort put into the structure and flow of this section. Out of curiosity, could you share a bit about your approach or tools used when drafting this content? If you leveraged any AI tools or models (e.g., GPT-based models), it would be interesting to know how they influenced your process.

Reply:
Hey I've changed and updated it , please check now, I apologise for the previous error; yes, I did take help from GPT- 4